Donation-platform/src/main/webapp/trendingScript.js

```diff
@@ -54,7 +57,8 @@ function updateResultsOnPage(charities) {
   //resultsContainer.innerHTML += '<li>' + 'name' + '</li>';
   // add results to the page
   //for (const name of charities) {
-  for (index = 0; index < charities.length; index++) {
-    resultsContainer.innerHTML += '<li>' + charities[index] + '</li>';
+  for (index = 0; index < charities.length - 1; index += 2) {
+    resultsContainer.innerHTML += '<li>' + charities[index] + ': ' + charities[index + 1] + '</li>';
```

Comment:
are we showing 2 charities per row? Please attach screenshot to make it clear.

Reply:
displaying "charity name: charity tags". attached a screenshot in a separate comment below.

Reply:
Then this should be a pair of <charity_name, charity_tags> and you should iterate over pair to fetch. This way of storing it can lead to problems.

Reply:
good point, fixed that and re-committed.